app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :users_id
+  has_many :likes, foreign_key: :users_id
+
```

Comment:
- [ ] Great job so far, in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) the association between the user and a comment is through the author_id column(foreign_key), kindly fix that and also for the like as well let's use the author_id.

<img width="701" alt="image" src="https://user-images.githubusercontent.com/59416136/228924367-028cabfe-63c7-4cc8-937b-fcee64e4249f.png">